op-batcher/batcher/espresso.go

```diff
@@ -158,25 +159,33 @@ func (l *BatchSubmitter) espressoBatchLoadingLoop(ctx context.Context, wg *sync.
 				continue
 			}
 
-			var batch_ptr *espresso.EspressoBatchI
+			var batch *derive.EspressoBatch
 
 			for {
 
-				batch_ptr = streamer.Next(ctx)
-				var batch = *batch_ptr
+				batch = streamer.Next(ctx)
+
 				if batch == nil {
 					break
 				}
 
+				// l.blockMutex.Lock()
+				// block := l.blocks[batch.Hash()]
+				// l.blockMutex.Unlock()
+
+				// if block == nil {
+				// 	panic("Not our block")
+				// }
+
 				// This should happen ONLY if the batch is malformed. BatchToIncompleteBlock has to guarantee
```

Comment:
Nit: Should `BatchToIncompleteBlock` be `ToBlock`?

Reply:
Addressed in https://github.com/EspressoSystems/optimism-espresso-integration/pull/96/commits/558e0e9f923cce1b19bb9e5ce7621c52a90a55cf.